cob_controller_configuration_gazebo/launch/robots/default_controllers_raw3-2.launch

```diff
@@ -35,7 +35,7 @@
 	<!-- start additional components -->
 	<include file="$(find cob_bringup)/drivers/sound.launch" />
 	<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
-		<arg name="robot" value="raw3-2"/>
+		<arg name="robot" value="$(arg robot)"/>
```

Comment:
please be so kind and apply this "fix" to all other raw's, too...for consistency...